docs/reference/data-streams/lifecycle/apis/delete-global-retention.asciidoc

```diff
@@ -0,0 +1,121 @@
+[[data-streams-delete-global-retention]]
+=== Deletes the global retention of data streams
+++++
+<titleabbrev>Deletes Data Stream Global Retention</titleabbrev>
+++++
+
+preview::[]
+
+Deletes the global retention settings that apply on every data stream managed by data stream lifecycle.
```

Comment:
I think it would be useful in all 3 of these to have a link to https://www.elastic.co/guide/en/elasticsearch/reference/current/data-stream-lifecycle.html here since this isn't a child of that page.

Reply:
Added.